eslint.config.js

```diff
@@ -130,6 +130,7 @@ export default [
 				QUnit: false,
 				url: true,
 				compareVersions: true,
+				jQueryVersionSince: false,
```

Comment:
Why wasn't this an issue before?

Reply:
This API was not used in `test/data/*` files previously.